src/components/Input.tsx

```diff
@@ -5,30 +5,49 @@ const Input = ({
   placeholder,
   helpText,
   disabled,
+  className,
+  value,
+  onChange,
+  required,
+  optional = false,
 }: {
   type: string;
   name: string;
   label: string;
   placeholder?: string;
   helpText?: string;
   disabled?: boolean;
+  className?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  required?: string;
+  optional?: boolean;
```

Comment:
aren't optional & required point to the same thing?

Reply:
No. required and optional are two different props. optional indicates that this field is optional and Required indicates that this field is mandatory. Optional props were needed because optional is written in front of some fields that this field is optional.

Reply:
> Pptional indicates that this field is optional and Required indicates that this field is mandatory

If a field is not optional then it is mandatory! So I don't think we need two fields to get the same value. I can just do `!optional` to get required check or just `!required` to get optional check. Not both.